wcfsetup/install/lang/de.xml

```diff
@@ -563,7 +563,7 @@
 		<item name="wcf.acp.devtools.project.instruction.value"><![CDATA[Datei(en)]]></item>
 		<item name="wcf.acp.devtools.project.instruction.value.description"><![CDATA[Die angegebene Datei wird während der Installation/des Updates verarbeitet werden.]]></item>
 		<item name="wcf.acp.devtools.project.instructions.description"><![CDATA[Die Anweisungen erklären der Installations- und Updateroutine, was gemacht werden muss, damit das Paket erfolgreich installiert/aktualisiert werden kann.]]></item>
-		<item name="wcf.acp.devtools.project.packageIdentifier.description"><![CDATA[Der Paketbezeichner identifiziert das Paket eindeutig. Der Bezeichner besteht aus mindestens drei durch Punkte getrennte Abschnitten. Jeder Abschnitt darf nur folgende Zeichen einhalten: <kbd>[A-z0-9-_]</kbd>.]]></item>
+		<item name="wcf.acp.devtools.project.packageIdentifier.description"><![CDATA[Der Paketbezeichner identifiziert das Paket eindeutig. Der Bezeichner besteht aus mindestens drei durch Punkte getrennte Abschnitte. Jeder Abschnitt darf nur folgende Zeichen einhalten: <kbd>[A-z0-9-_]</kbd>.]]></item>
```

Comment:
I agree that the previous version is incorrect, but I believe the correct fix is this:
```suggestion
		<item name="wcf.acp.devtools.project.packageIdentifier.description"><![CDATA[Der Paketbezeichner identifiziert das Paket eindeutig. Der Bezeichner besteht aus mindestens drei durch Punkte getrennten Abschnitten. Jeder Abschnitt darf nur folgende Zeichen einhalten: <kbd>[A-z0-9-_]</kbd>.]]></item>
```